lib/adLDAP/adLDAP.php

```diff
@@ -682,14 +689,24 @@ public function close() {
     * @param string $password A user's AD password
     * @param bool optional $preventRebind
     * @return bool
+    * @throws adLDAPException
     */
     public function authenticate($username, $password, $preventRebind = false) {
+    	/** @noinspection PhpUndefinedClassInspection */
+		Logging::var_trace('username', $username);
+		/** @noinspection PhpUndefinedClassInspection */
+		Logging::var_trace('password', $password);

```

Comment:
I would think we would want to strictly avoid logging user passwords in plaintext.